advanced03-AF_XDP/af_xdp_user.c

```diff
@@ -605,7 +627,7 @@ int main(int argc, char **argv)
 	/* Cleanup */
 	xsk_socket__delete(xsk_socket->xsk);
 	xsk_umem__delete(umem->umem);
-	xdp_link_detach(cfg.ifindex, cfg.xdp_flags, 0);
+	xdp_program__detach(prog, cfg.ifindex, cfg.xdp_flags, 0);
```

Comment:
At the moment - exit leaves the program loaded on the interface as the exit is handled in exit_application()...
Should we clean up?

Reply:
I think we should clean up, yes. Same for the other tutorials where user space prog is coupled to the BPF prog. I need to check the tracing examples do. The basic examples just have a loader so it's expected to leave the prog loaded.